packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -539,6 +543,27 @@ export class Viewer implements IDisposable {
         await this._updateModel(undefined, undefined, abortSignal);
     }
 
+    /**
+     * Show the inspector for this scene
+     * @param rootElement the root element to attach the inspector to. defaults to body (but body will need to be styled correctly)
+     */
+    public async showInspector(rootElement?: HTMLElement): Promise<void> {

```

Comment:
We should move this to the viewerElement layer, but I can take that on and push an update to this PR.